Update manage-user-profiles.md (#2077)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/manage-user-profiles.md

    ![Manage the owners of a OneDrive](media/120f7c8c-262f-4a41-a484-e830c662f534.png)
  
For info about automatically transferring ownership of OneDrive to a user's manager when the user account is marked for deletion, see [Set up access delegation](/onedrive/retention-and-deletion).

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> >For full OneDrive for Business functionality the user must be a site collection administrator of their OneDrive for Business site. It is not supported to remove this permission.

Reviewer: ```suggestion
> For full OneDrive functionality, the user must be a site collection administrator of the organization's OneDrive site. Removing this permission is not supported.
```

Reviewer: Please do not use OneDrive for Business.
